.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.acceptSuggestionOnEnter": "on"
```

Comment:
not needed, because this is just for users, who are using vscode. to prevent commit this file, you could add 

`/.vscode` to `.gitignore` file

Reply:
please remove this file from commit